N/nlminb/build_tarballs.jl

```diff
@@ -0,0 +1,60 @@
+using BinaryBuilder
+
+name = "nlminb"
+version = v"0.1.0"
+
+# Collection of sources required to build NLopt
+sources = [
+    GitSource("https://github.com/geo-julia/nlminb.f.git",
+              "6b2d460b866911e8bfca32b1217c08fb78c3111f"), # v0.1.0
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/nlminb.f
+mkdir build 
+cd build
+
+if [[ "${nbits}" == 64 ]] && [[ "${target}" != aarch64* ]]; then
+    OPENBLAS="${libdir}/libopenblas64_.${dlext}"
+else
+    OPENBLAS="${libdir}/libopenblas.${dlext}"
+fi
+
+# -DBLAS_LIBRARIES=${libdir}/libopenblas.${dlext}
+CMAKE_FLAGS+=(-DCMAKE_INSTALL_PREFIX=${prefix}
+              -DBLAS_LIBRARIES=${OPENBLAS}
+              -DCMAKE_BUILD_TYPE=Release
```

Comment:
https://docs.binarybuilder.org/stable/build_tips/#CMake-builds
```suggestion
              -DCMAKE_BUILD_TYPE=Release
              -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN}
```